### PR TITLE
Change pass page 95%

### PR DESCRIPTION
**Pull request by Ajknight121**

### CODE REVIEW

**Comment by Ajknight121** [comment]
So the remaining issue is the Password input does not show a default text of "password" that is visible to the user. I also was not able to figure out how to make the background image fill the page while not activating the scroll bar.

File: src/pages/ChangePassword.tsx [hunk 1]
```diff
@@ -18,13 +21,23 @@ export default function ChangePassword(): JSX.Element {
...
-    </div>)
+    <body>
+        <div id="change-password">
+            
+            <div id = "login">
+                <div id = "img">
+                <img src={logo}/><br/>
+                </div>
+                <h1>Change password</h1><br/>
+                <label><h2>Username:</h2></label>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:32]: Remove the labels they should be inside the textbooks

File: src/pages/ChangePassword.tsx [hunk 2]
```diff
@@ -18,13 +21,23 @@ export default function ChangePassword(): JSX.Element {
...
+                <input type="text" defaultValue={"Username"} onChange = {event => username = event.target.value}/><br/>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:33]: Change the defaultValue param to placeholder without the javascript injection

File: src/pages/ChangePassword.tsx [hunk 3]
```diff
@@ -18,13 +21,23 @@ export default function ChangePassword(): JSX.Element {
...
+                <label><h2>New password:</h2></label>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:34]: See above

File: src/pages/ChangePassword.tsx [hunk 4]
```diff
@@ -18,13 +21,23 @@ export default function ChangePassword(): JSX.Element {
...
+                <input type="password" defaultValue={"password"} onChange = {event => newpass = event.target.value}/><br/>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:35]: See above

File: src/pages/ChangePassword.tsx [hunk 5]
```diff
@@ -2,6 +2,9 @@ import { useSelector } from "react-redux";
 import { apiReducerState } from "../redux/reducers/apikey";
 import API from "../util/Api";
 import { useSearchParams } from "react-router-dom";
+import "./ChangePassword.scss";
+import background from "../img/people.jpeg";
+import logo from "../img/logo.png";
 
 export default function ChangePassword(): JSX.Element {
     let username: string = ""
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:10]: Finish the functionality here

File: src/pages/ChangePassword.tsx [hunk 6]
```diff
@@ -18,13 +21,23 @@ export default function ChangePassword(): JSX.Element {
...
+            <div id = "login">
+                <div id = "img">
+                <img src={logo}/><br/>
+                </div>
+                <h1>Change password</h1><br/>
+                <label><h2>Username:</h2></label>
+                <input type="text" defaultValue={"Username"} onChange = {event => username = event.target.value}/><br/>
+                <label><h2>New password:</h2></label>
+                <input type="password" defaultValue={"password"} onChange = {event => newpass = event.target.value}/><br/>
+                <div className="button" onClick={ChangePass}>Submit</div>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:36]: Change to ChangePassword

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Make sure that changes on a branch directly relate to the stated changes

- [x] Fix the issues with the textbox sizes
- [ ] Fix placeholder changes
- [ ] Remove the labels

File: src/pages/ChangePassword.tsx [hunk 7]
```diff
@@ -21,23 +21,20 @@ export default function ChangePassword(): JSX.Element {
...
                 <div id = "img">
                 <img src={logo}/><br/>
                 </div>
                 <h1>Change password</h1><br/>
-                <label><h2>Username:</h2></label>
-                <input type="text" defaultValue={"Username"} onChange = {event => username = event.target.value}/><br/>
-                <label><h2>New password:</h2></label>
-                <input type="password" defaultValue={"password"} onChange = {event => newpass = event.target.value}/><br/>
+                {/* <label><h2>Username:</h2></label> */}
+                <input type="text" placeholder={"Username"} onChange = {event => username = event.target.value}/><br/>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:32]: This should be "New Password"

File: src/pages/ChangePassword.tsx [hunk 8]
```diff
@@ -21,23 +21,20 @@ export default function ChangePassword(): JSX.Element {
...
+                {/* <label><h2>New password:</h2></label> */}
+                <input type="password" placeholder={"New password"} onChange = {event => newpass = event.target.value}/><br/>
```
Comment (ChristianBingman) [inline src/pages/ChangePassword.tsx:34]: This should be "New Password Confirm"

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
- [ ] Change the placeholder to what is commented

**Pull request merged**